### PR TITLE
Feature/ab#65488 publish ocelot to kenga

### PR DESCRIPTION
**Pull request by TCPrimedPaul**
Enforce https when communicating with mpdis wrapper;
Add request/response logging;
Configure application settings for environments;
Redirect dashboard to kenga.

### CODE REVIEW

File: src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-dev.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "DownstreamScheme": "https",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "//pokeapi.co",
+          "Port": "443"
+        }
+      ],
+      "UpstreamPathTemplate": "/pokemon",
+      "UpstreamHttpMethod": [ "Get" ]
+    },
```
Comment (AlexandreSanscartier) [inline src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-dev.json:20]: Can remove the pokemon test api.
```suggestion
```

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
Great job! Only thing I would say is remove the pokemon references. Works without VPN now, which is great.

File: src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-dev.json (hunk 1 above)
Comment (TCPrimedPaul) [reply src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-dev.json]: Pokemon is pokegone

File: src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-acc.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      "DownstreamScheme": "https",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "//pokeapi.co",
+          "Port": "443"
+        }
+      ],
+      "UpstreamPathTemplate": "/pokemon",
+      "UpstreamHttpMethod": [ "Get" ]
+    },
```
[the remark this answers is not in the document]
Comment (TCPrimedPaul) [reply src/CSF.APIM/CSF.APIM.Gateway/Ocelot.Acceptance-acc.json]: Pokemon is pokegone

**Pull request merged**